src/lib/errors/error-handler.ts

```
@@ -0,0 +1,166 @@
+}
+
+/**
+ * Format Prisma errors into standardized format
+ */
+function formatPrismaError(error: Prisma.PrismaClientKnownRequestError): ErrorResponse {
+  const timestamp = new Date().toISOString();
+
+  switch (error.code) {
+    case 'P2002':
+      // Unique constraint violation
+      const target = error.meta?.target as string[] | undefined;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)